src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+import { useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+import { PeopleFilters } from '../components/PeopleFilters';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+
+  const [searchParams] = useSearchParams();
+
+  const loadPeople = () => {
+    setIsLoading(true);
+    setError('');
+
+    getPeople()
+      .then(res => {
+        const peopleWithFamily = res.map(person => {
+          const father = res.find(p => p.name === person.fatherName);
+          const mother = res.find(p => p.name === person.motherName);
+
+          return {
+            ...person,
+            father,
+            mother,
+          };
+        });
+
+        setPeople(peopleWithFamily);
+      })
+      .catch(() => setError('Unable to load data'))
+      .finally(() => setIsLoading(false));
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || null;
```

Comment:
The `sex` parameter is set to `null` if not present, which might not be ideal for filtering logic. Consider using an empty string or a default value that aligns with your filtering logic to avoid potential issues.